src/ucp/rndv/proto_rndv.c

```diff
@@ -925,8 +941,11 @@ ucp_proto_rndv_handle_rtr(void *arg, void *data, size_t length, unsigned flags)
     /* RTR covers the whole send request - use the send request directly */
     ucs_assert(req->flags & UCP_REQUEST_FLAG_PROTO_INITIALIZED);
 
-    select_param = &req->send.proto_config->select_param;
-    op_attr_mask = ucp_proto_select_op_attr_unpack(select_param->op_attr);
+    select_param  = &req->send.proto_config->select_param;
+    op_attr_mask  = ucp_proto_select_op_attr_unpack(select_param->op_attr);
+    /* Remove FAST_CMPL since it should be considered only for one stage
+     * protocols (like eager) */
+    op_attr_mask &= ~UCP_OP_ATTR_FLAG_FAST_CMPL;
```

Comment:
```suggestion
    /* Remove FAST_CMPL since it should be considered only for one stage
     * protocols (like eager) */
    op_attr_mask  = ucp_proto_select_op_attr_unpack(select_param->op_attr) &
                    ~UCP_OP_ATTR_FLAG_FAST_CMPL;
```